src/senselab/audio/tasks/voice_cloning/api.py

```diff
@@ -17,27 +15,39 @@ def clone_voices(
     device: Optional[DeviceType] = None,
     **kwargs: Any,  # noqa:ANN401
 ) -> List[Audio]:
-    """Clones voices from source audios to target audios using the given model."""
+    """Clones voices from source audios to target audios using the given model.
+
+    Args:
+        source_audios (List[Audio]): A list of source audio samples.
+        target_audios (List[Audio]): A list of target audio samples.
+        model (SenselabModel, optional): The model to use for voice cloning.
+            Defaults to TorchModel(path_or_uri="bshall/knn-vc", revision="master").
```

Comment:
We should specify in the documentation itself that as of now, only KNNVC is supported.